include/pros/apix.h

```diff
@@ -531,12 +531,21 @@ int32_t fdctl(int file, const uint32_t action, void* const extra_arg);
  */
 #define DEVCTL_FIONREAD 16
 
+/**
+ * Action macro to check if there is space available in the Generic Serial
+ * Device's output buffer
+ *
+ * The extra argument is not used with this action, provide any value (e.g.
+ * NULL) instead
+ */
+#define DEVCTL_FIONWRITE 17
+
 /**
  * Action macro to set the Generic Serial Device's baudrate.
  *
  * The extra argument is the baudrate.
  */
-#define DEVCTL_SET_BAUDRATE 17
+#define DEVCTL_SET_BAUDRATE 18
```

Comment:
As annoying as it sounds, I'd prefer to leave `DEVCTL_SET_BAUDRATE` as 17 and have `FIONWRITE` be 18.

Reply:
Ok, that makes sense. I've changed `DEVCTL_SET_BAUDRATE` back to 17, though, I left it after `DEVCTL_FIONWRITE`. I figure that ordering makes more sense even though numerically it's out of order.